File: Pemrograman1/Modul4/Python/PRAK404-2310817310009-MuhammadFauzanAhsani.py
while True:
    print('''
Pilih program
1. Penjumlahan
2. Pengurangan
3. Perkalian
4. Pembagian
5. Exit
''', end = "")
    selector = int(input("Masukkan pilihan : "))

    if (selector < 5):
        val1 = int(input("Masukkan nilai pertama :"))
        val2 = int(input("Masukkan nilai kedua : "))

        if (selector == 1):
            print(f"Hasil penjumlahan antata {val1:.2f} dengan {val2:.2f} adalah {(val1 + val2):.2f} ")

        elif (selector == 2):
            print(f"Hasil pengurangan antata {val1:.2f} dengan {val2:.2f} adalah {(val1 - val2):.2f} ")

        elif (selector == 3):
            print(f"Hasil perkalian antata {val1:.2f} dengan {val2:.2f} adalah {(val1 * val2):.2f} ")

        elif (selector == 4):
            print(f"Hasil pembagian antata {val1:.2f} dengan {val2:.2f} adalah {(val1 / val2):.2f} ")

    elif (selector == 5):
        print("Terimakasih, telah menggunakan kalkulator Muhammad Fauzan Ahsani")
        break
    else:
        print("Input anda salah, silahkan coba lagi")

File: Pemrograman1/Modul4/C/PRAK404-2310817310009-MuhammadFauzanAhsani.c
#include <stdio.h>

int main(){
    int selector;
    double val1, val2;

    while(1){
    printf("Pilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\nMasukkan Pilihan : ");
    scanf("%d", &selector);

    if(selector < 5){
        printf("Masukkan nilai pertama : ");
        scanf("%lf",&val1);
        printf("Masukkan nilai kedua : ");
        scanf("%lf", &val2);
        if(selector == 1){
            printf("Hasil penjumlahan antata %.2lf dengan %.2lf adalah %.2lf\n", val1, val2, val1 + val2);
        }
        else if (selector == 2){
            printf("Hasil pengurangan antata %.2lf dengan %.2lf adalah %.2lf\n", val1, val2, val1 - val2);
        }
        else if (selector == 3){
            printf("Hasil perkalian antata %.2lf dengan %.2lf adalah %.2lf\n", val1, val2, val1 * val2);
        }
        else if (selector == 4){
            printf("Hasil pembagian antata %.2lf dengan %.2lf adalah %.2lf\n", val1, val2, val1 / val2);
        }
    }
    else if (selector == 5){
        printf("Terimakasih, telah menggunakan kalkulator Muhammad Fauzan Ahsani");
        break;
    }
    else {
        printf("Input anda salah, silahkan coba lagi\n");
    }
    }
    return 0;
}
